packages/core/src/node/mdx/options.ts

```diff
@@ -83,6 +86,10 @@ export async function createMDXOptions(
       ],
       ...rehypePluginsFromConfig,
       ...rehypePluginsFromPlugins,
-    ],
+      // This is what permits to embed HTML elements with format 'md'
+      // See https://github.com/facebook/docusaurus/pull/8960
+      // See https://github.com/mdx-js/mdx/pull/2295#issuecomment-1540085960
+      format === 'md' && [rehypeRaw, { passThrough: nodeTypes }],
```

Comment:
As I mentioned, would this work for 3rd-party plugins which expects `rehype-raw` to be used first?